File: betam_app/lib/models/radio_station.dart
class RadioStation {
  final String id;
  final String name;
  final String frequency;
  final String city;
  final String imageUrl;
  final String? streamUrl;

  RadioStation({
    required this.id,
    required this.name,
    required this.frequency,
    required this.city,
    required this.imageUrl,
    this.streamUrl,
  });
}

File: betam_native/app/src/main/java/com/example/ethiopianradio/data/model/RadioStation.kt
package com.example.ethiopianradio.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class RadioStation(
    val id: String = "",
    val name: String = "",
    val frequency: String = "",
    val city: String = "",
    val imageUrl: String = "",
    val streamUrl: String = ""
) : Parcelable

